idr0045-study.txt

```diff
@@ -0,0 +1,54 @@
re02 - Zygotes isolated from EGFP-Tuba C57BL/6J x C3H/He F1 females mated with C57BL/6J x C3H/He F1 were and fixed and stained as described (https://doi.org/10.1101/198275). Embryos were imaged on a SP8 Leica confocal microscope equipped with a 63* C-Apochromat 1.2 NA water immersion objective lens and acquired at 80 nm in xy and 320 nm in z.	SupplementaryFigure05 - Zygotes were isolated from C57BL/6J x C3H/He F1 females mated with C57BL/6J x C3H/He F1 males and injected with EB3-EGFP. Zygotes were imaged with 800 msec time resolution on an inverted light sheet microscope (doi: 10.1038/nmeth.3690) at 130nm xy and 130 nm z resolution.	SupplementaryFigure06A - Zygotes were isolated from H2B-mCherry C57BL/6J x C3H/He F1 females mated with Mus spretus males and injected with TALE-mClover and TALE-tdiRFP670 and imaged with 60 sec time resolution on an inverted light sheet microscope (doi: 10.1038/nmeth.3690) at 130nm xy and 520nm z resolution using 488, 561 and 642 laser lines. 	SupplementaryFigure06H - Zygotes were isolated from C57BL/6J x C3H/He F1 females mated to C57BL/6J x C3H/He F1 males mice that were supplied continuously for 6 weeks with drinking water containing 0.5 mg/ml 5-ethynyl-2'-deoxyuridine (EdU). Paternal chromatin was labelled with Alexa Fluor 647 and the whole nucleus counter stained with 100 nM Sytox Green (Molecular probes S7020). Embryos were imaged on an inverted light sheet microscope (doi: 10.1038/nmeth.3690) at 130nm xy and 520nm z resolution using 488 and 642 laser lines. 	SupplementaryFigure08 - Zygotes were isolated from C57BL/6J x C3H/He F1 females mated with Mus spretus males and injected with TALE-mClover, TALE-tdiRFP670 and EB3-mEGFP and imaged with 60 sec time resolution on an inverted light sheet microscope (doi: 10.1038/nmeth.3690) at 130nm xy and 520nm z resolution using 488, 561 and 642 laser lines.	SupplementaryFigure09 - Zygotes were isolated from C57BL/6J x C3H/He F1 females mated with C57BL/6J x C3H/He F1 males and fixed and stained as described (https://doi.org/10.1101/198275). Embryos were imaged on a SP8 Leica confocal microscope equipped with a 63* C-Apochromat 1.2 NA water immersion objective lens. Images of embryos were acquired at 90 nm in xy and 360 nm in z.	SupplementaryFigure10A - Zygotes were isolated from C57BL/6J x C3H/He F1 females mated with C57BL/6J x C3H/He F1 males and fixed and stained as described (https://doi.org/10.1101/198275). Embryos were imaged on a SP8 Leica confocal microscope equipped with a 63* C-Apochromat 1.2 NA water immersion objective lens. Images of embryos were acquired at 90 nm in xy and 360 nm in z.	SupplementaryFigure10B - Zygotes were isolated from C57BL/6J x C3H/He F1 females mated with C57BL/6J x C3H/He F1 males and fixed and stained as described (https://doi.org/10.1101/198275). Embryos were imaged on a SP8 Leica confocal microscope equipped with a 63* C-Apochromat 1.2 NA water immersion objective lens. Images of embryos were acquired at 90 nm in xy and 360 nm in z.
```

Comment:
Only the first part is put as the project description. The rest should probably be moved to `Experiment Comments`